src/main/java/org/liquidengine/legui/demo/ExampleGui.java

```diff
@@ -249,7 +250,7 @@ public ExampleGui(int width, int height) {
         Button turnWidVisible = new Button("", 360, 280, 20, 20);
         turnWidVisible.getListenerMap().addListener(MouseClickEvent.class, (MouseClickEventListener) event -> {
             if (CLICK == (event.getAction())) {
-                widget.setVisible(true);
+                widget.getStyle().setDisplay(Style.DisplayType.MANUAL);
```

Comment:
same here - display `FLEX`  (for all widgets)
The reason is that display `FLEX` show how that container becomes flexible (used this as guide https://www.w3schools.com/css/css3_flexbox.asp)